src/gui/folderstatusmodel.cpp

```diff
@@ -269,131 +238,112 @@ QVariant FolderStatusModel::data(const QModelIndex &index, int role) const
-        case Columns::HeaderRole: {
-            if (auto *space = getSpace()) {
-                return space->displayName();
-            }
-            return f->displayName();
+        return tr("Local folder: %1").arg(f->shortGuiLocalPath());
+    }
+    case Columns::FolderErrorMsg: {
+        auto errors = f->syncResult().errorStrings();
+        const auto legacyError = FolderMan::instance()->unsupportedConfiguration(f->path());
+        if (!legacyError) {
+            errors.append(legacyError.error());
         }
-        case Columns::FolderImage:
-            if (auto *space = getSpace()) {
-                return space->image();
-            }
-            return Resources::getCoreIcon(QStringLiteral("folder-sync"));
-        case Columns::FolderSyncPaused:
-            return f->syncPaused();
-        case Columns::FolderAccountConnected:
-            return accountConnected;
-        case Columns::FolderStatusIconRole: {
-            auto status = f->syncResult();
-            if (!accountConnected) {
-                status.setStatus(SyncResult::Status::Offline);
-            } else if (f->syncPaused() || f->accountState()->state() == AccountState::PausedDueToMetered) {
-                status.setStatus(SyncResult::Status::Paused);
-            }
-            return Theme::instance()->syncStateIconName(status);
+        if (f->syncResult().hasUnresolvedConflicts()) {
+            errors.append(tr("There are unresolved conflicts. Click for details."));
         }
-        case Columns::SyncProgressItemString:
-            return progress._progressString;
-        case Columns::WarningCount:
-            return progress._warningCount;
-        case Columns::SyncProgressOverallPercent:
-            return progress._overallPercent;
-        case Columns::SyncProgressOverallString:
-            return progress._overallSyncString;
-        case Columns::FolderSyncText: {
-            if (auto *space = getSpace()) {
-                if (!space->drive().getDescription().isEmpty()) {
-                    return space->drive().getDescription();
-                }
+        if (f->isReady() && f->virtualFilesEnabled() && f->vfs().mode() == Vfs::Mode::WithSuffix) {
+            errors.append({
+                tr("The suffix VFS plugin is deprecated and will be removed in the 7.0 release.\n"
+                   "Please use the context menu and select \"Disable virtual file support\" to  ensure future access to your synced files.\n"
+                   "You are going to lose access to your sync folder if you do not do so!"),
+            });
+        }
+        return errors;
+    }
+    case Columns::DisplayName: {
+        if (auto *space = getSpace()) {
+            return space->displayName();
+        }
+        return f->displayName();
+    }
+    case Columns::FolderImageUrl:
+        if (f->accountState()->supportsSpaces()) {
+            // TODO: the url hast random parts to enforce a reload
```

Comment:
Using the random number is a hack.
In theory, we could directly use a http url.
However, we would need to provide a new networkaccess manager as the qml context runs in a different thread.
Our current nam, used for authentication and logging, is currently not designed in a way that really supports multiple instances.
https://github.com/owncloud/client/blob/1eeef49749f78eb3ec4621e4869166b720b12c29/src/libsync/creds/httpcredentials.cpp#L134-L134